src/utils/fetchClient.ts

```diff
@@ -1,6 +1,6 @@
 const BASE_URL = 'https://mate.academy/students-api';
 
-// a promise resolved after a given delay
+// returns a promise resolved after a given delay
```

Comment:
don't change this file

Reply:
on my PC this file not changet)) 
<img width="780" alt="image" src="https://user-images.githubusercontent.com/104455381/202852872-db022600-c091-41e6-93a3-df1d04f4f645.png">